### ISSUE TITLE
Game error - access violation 0xc0000005

### ISSUE THREAD

**Issue by alexphilip0019** [Lyall/FFXVIFix#59]
Using version 0.8.0 

when running the game as administrator getting error 
![image](https://github.com/user-attachments/assets/3750ef4a-02ef-4d7d-b3b3-413a9ba1d12d)

**Comment by alexphilip0019** [comment]
[FFXVIFix.log](https://github.com/user-attachments/files/17084996/FFXVIFix.log)

adding log file if needed

**Comment by Lyall** [comment]
Why do you need to run the game as Admin? Does this error happen without that?

**Comment by alexphilip0019** [comment]
It happens when running not as admin as well

**Comment by Lyall** [comment]
Does it happen without FFXVIFix installed?

**Comment by alexphilip0019** [comment]
No

**Comment by Lyall** [comment]
Hmm, it's a pretty generic error so I'm not sure what to suggest. You could try renaming `dinput8.dll` to something different like `version.dll` and see if that works. You could also try updating your [Visual C++ libraries](https://github.com/abbodi1406/vcredist/releases). 

Has any version of FFXVIFix worked for you before or is this your first time using it?

**Comment by alexphilip0019** [comment]
Having same error with older versions as well, rename to version.dll also not working

**Comment by Dakusukai** [comment]
Got the same error but while playing I got it at random times, I'm gonna try to go back do 0.7.9 since I haven't had any problem before (I've played with almost all version starting from 0.7.5) to see if it's on my end or not

**Comment by alexphilip0019** [comment]
I got it working, needed to disable amd fsr 3 frame generation setting.
@Lyall can any thing be done regarding this like when game starts with fix it will disable frame generation if it is enabled and enable to back on after game started

**Comment by Lyall** [comment]
> I got it working, needed to disable amd fsr 3 frame generation setting. @Lyall can any thing be done regarding this like when game starts with fix it will disable frame generation if it is enabled and enable to back on after game started

That's odd. Could you maybe try setting the cutscene frame generation option to false in `FFXVIFix.ini`? Other than that setting, nothing else should interfere with frame generation.

**Comment by IronSlasher** [comment]
I reverted to 0.7.9 as I am encountering the same issue at random times.

**Comment by alexphilip0019** [comment]
I think frame generation on whole is issue if ffxvi fix is removed and frame generation is in sometimes also getting error operation completed successfully

**Comment by Lyall** [comment]
Hmm, you're not using any kind of overlays or anything? Or software like Razer Synapse or iCUE or rubbish like that? Starting to think this is more of game/system issue than something caused directly by FFXVIFix (though I could be wrong).

**Comment by alexphilip0019** [comment]
No overlays or extra software
Ffxvi fix is working perfectly fine when no frame generation is used. Let's wait for game updates to see if issue gets resolved

**Comment by WaffleWasher** [comment]
I've been having these crashes, too. Pretty much since the full game came out, paired with the fix. Never experienced it in the demo. I installed 0.8.1 this morning (been keeping up with the fix releases), Played through all of Caer Moore (Benedikta dungeon) without issues. Then realized HUD was still at 16:9, changed the INI setting to 0 (43:18 monitor), and the crashes came back up. Not sure if it's coincidence or not. Tried it with and with and without DoF disabled on previous versions of the fix. I do not use FG, just DLAA. 

This is tough to figure out without a proper error message. I did get another error in English a few times stating something along the lines of "Access violation trying to access invalid memory parameters" Kaldaien has also touched on this in the Steam forms. 

Also, should note, this did not happen before updating nVidia drivers to the latest driver, the most recent version of drivers did not present this with the fix, but I am unable to determine if that is the issue or not. I'll try a roll back on drivers, check my libraries and see if that leads to stability.

**Comment by Dakusukai** [comment]
I have this problem, sometimes it crashes at random time while playing with a Ryzen 7 CPU so don't know. And didn't had any problem in my first 8h so I don't know, tried going back to 0.7.9 but I got it again too so I don't know. I'll wait for a game patch I think since it's seems to be a game problem

**Comment by Lyall** [comment]
The 0xC5 error is very vague, it literally just means the game tried to access a part of memory that wasn't correct so it can be caused by all manners of different issues. I'm more inclined to believe this is a stability issue or a game issue given that myself and the majority of people out there have not encountered this.

**Comment by Eve-Knight** [comment]
I've rarely had this a couple times, including before I knew this mod existed. So it's a base game issue.

**Comment by VagrantZero** [comment]
This problem was so bad for me it basically broke the game in a specific spot and made me unable to leave The Hideaway. I was able to fix it by deleting my shader cache and force the game to recompile it at startup. I don't believe this error is tied to the fix as my game was hardlocking in the exact same spot even with the fix disable.

TLDR: delete your shader cache. It's located at C:\Users\[username]\AppData\Local\SquareEnix\FINAL FANTASY XVI

**Comment by nlikeflynn** [comment]
I’ve used FSR and FG the whole game and never had a crash. I decided to put my HUD at 0 which is span screen and had 3 crashes within 30 minutes. I changed the option back to 4 which is 21:9 and everything went back to normal. I’m on v8.0.

**Comment by Lyall** [comment]
> I’ve used FSR and FG the whole game and never had a crash. I decided to put my HUD at 0 which is span screen and had 3 crashes within 30 minutes. I changed the option back to 4 which is 21:9 and everything went back to normal. I’m on v8.0.

What resolution are you playing the game at? It's unlikely this part of the fix would cause issues, the code for it only runs during game startup, opening the pause menu and resolution changes.

**Comment by nlikeflynn** [comment]
> > I’ve used FSR and FG the whole game and never had a crash. I decided to put my HUD at 0 which is span screen and had 3 crashes within 30 minutes. I changed the option back to 4 which is 21:9 and everything went back to normal. I’m on v8.0.
> 
> What resolution are you playing the game at? It's unlikely this part of the fix would cause issues, the code for it only runs during game startup, opening the pause menu and resolution changes.

I’m at 3440x1440 FSR Balanced with FG

**Comment by WaffleWasher** [comment]
> The 0xC0000005 error is very vague, it literally just means the game tried to access a part of memory that wasn't correct so it can be caused by all manners of different issues. I'm more inclined to believe this is a stability issue or a game issue given that myself and the majority of people out there have not encountered this.

To add to this:

I installed the newest Asus BIOS for my board for my 14900k, and since I've done that, I don't hit TJ Max and the OC for my GPU isn't being bottlenecked. So now UE4 shader compilations don't crash, and not once has FFXVI crashed, in fact I gained about 15FPS on average.

**Comment by WaffleWasher** [comment]
> > The 0xC0000005 error is very vague, it literally just means the game tried to access a part of memory that wasn't correct so it can be caused by all manners of different issues. I'm more inclined to believe this is a stability issue or a game issue given that myself and the majority of people out there have not encountered this.
> 
> To add to this:
> 
> I installed the newest Asus BIOS for my board for my 14900k, and since I've done that, I don't hit TJ Max and the OC for my GPU isn't being bottlenecked. So now UE4 shader compilations don't crash, and not once has FFXVI crashed, in fact I gained about 15FPS on average. I have been keeping up with each update of the fix, and used it since the demo.

**Comment by That-Ruben** [comment]
I get this error when running the game for the first time. I have to turn down performance core ration by 1 in XTU to prevent it. After the first time I can turn the perf core ratio back up. 
I had to do it again after a game update though
14900 things
